src/css/cs/items/_aa_planner.scss

```diff
@@ -16,6 +16,19 @@
   .uc-fluid-table table {
     font-size: 11px;
   }
+  
+  .ps_content .uc-fluid-table table table .ps_box-button a {
```

Comment:
@darrylumich does it work when you nest this under `.uc-fluid-table table`?

Something like:

```scss
.uc-fluid-table table {
  font-size: 11px;
  table .ps_box-button a {
    background: ...
  }
}
```